### ISSUE TITLE
what are EDL , EDLPS .... stand for ?

### ISSUE THREAD

**Issue by wangpeng138375** [dev-chauhan/PQG-pytorch#1]

**Comment by dev-chauhan** [comment]
These names are given to different variants of model used here for paraphrase generation tasks.
ED represents use of encoder-decoder network used in model
L represents model is trained using Cross-entropy loss 
P represents model is trained using [pair-wise discriminator loss](https://www.aclweb.org/anthology/C18-1230.pdf)
LP represents both above are used for training
S represents sharing of parameters of encoder from encoder-decoder network and discriminator used for pair-wise discriminator loss 
G represents use of discriminator to train decoder as generator using adversarial loss.
example :
EDLPS : encoder-decoder network with shared parameters of encoder and discriminator(module used for pair-wise discriminator loss) trained using both cross-entropy loss and pair-wise discriminator loss.

**Comment by wangpeng138375** [comment]
Thanks very much for your detail explanation.
